app/Http/Controllers/AdminController.php

```diff
@@ -11,8 +11,31 @@ class AdminController extends Controller
     public function getBasicAdminPage()
     {
         $allUsers = User::all();
+        $allUserCount = User::all()->count();
+        $user = Auth::user();
+        $canEmailCount = 0;
+        foreach($allUsers as $users){
+            if($users->can_contact === true)
+            {
+                $canEmailCount++;
+            }
+        }
```

Comment:
A more efficient way to do this would be:
`$canEmailCount = User::where('can_contact', true)->count();`
But the for loop is okay too!